bot.py

```diff
@@ -1,545 +1,755 @@
+        query = message.text.replace('/remcode ', '').strip()
+        if changer(query, 0, 'code.data') == True:
+            await message.reply(lng.getmess(message, "removed"), parse_mode = "Markdown")
+        else:
+            await message.reply(lng.getmess(message, "no_changes"), parse_mode = "Markdown")
+
+@dp.message(commands = ['addcode'])
+async def addcode_mess(message: types.Message):
+    if users.admin(message) == True and message.from_user.id == message.chat.id:
+        query = message.text.replace('/addcode ', '').strip()
+        if changer(query, 1, 'code.data') == True:
+            await message.reply(lng.getmess(message, "added"), parse_mode = "Markdown")
+        else:
+            await message.reply(lng.getmess(message, "no_changes"), parse_mode = "Markdown")
+
+@dp.message(commands = ['remcost'])
+async def remcost_mess(message: types.Message):
+    if users.admin(message) == True and message.from_user.id == message.chat.id:
+        query = message.text.replace('/remcost ', '').strip()
+        if changer(query, 0, 'cost.data') == True:
+            await message.reply(lng.getmess(message, "removed"), parse_mode = "Markdown")
+        else:
+            await message.reply(lng.getmess(message, "no_changes"), parse_mode = "Markdown")
+
+@dp.message(commands = ['addcost'])
+async def addcost_mess(message: types.Message):
+    if users.admin(message) == True and message.from_user.id == message.chat.id:
+        query = message.text.replace('/addcost ', '').strip()
+        if changer(query, 1, 'cost.data') == True:
+            await message.reply(lng.getmess(message, "added"), parse_mode = "Markdown")
+        else:
+            await message.reply(lng.getmess(message, "no_changes"), parse_mode = "Markdown")
+
+@dp.message(commands = ['listcode'])
+async def listcode_mess(message: types.Message):
+    if users.admin(message) == True and message.from_user.id == message.chat.id:
+        await message.reply(cost_code_reactor(message.text.replace('/listcode', '').strip(), 'code.data', full = True), parse_mode = "Markdown")
+
+@dp.message(commands = ['listcost'])
+async def listcode_mess(message: types.Message):
+    if users.admin(message) == True and message.from_user.id == message.chat.id:
+        await message.reply(cost_code_reactor(message.text.replace('/listcost', '').strip(), 'cost.data', full = True), parse_mode = "Markdown")
+
+@dp.message(commands = ['pw', 'pws'])
+async def search_mess(message: types.Message):
+    if users.getstatus(message) == Status.AUTH or users.getstatus(message) == Status.MAYDAY:
+        com = '/pws' if '/pws' in message.text else '/pw'
+        query = message.text.replace(com, '').replace(f'@{BOT_USERNAME}', '').strip()
+        if query == '':
+            await message.reply(lng.getmess(message, "empty"), parse_mode = "Markdown")
+        else:
+            try:
+                macs = [m.upper().replace('-', ':') for m in re.findall(MAC, query)]
```

Comment:
```suggestion
                macs = [m for m in re.findall(MAC, query)]
```
База сама понимает ['-', '.', ':'] в Мак адресах.